lib/ecto/postgres/pg_framerate_migrations.ex

```diff
@@ -172,39 +177,84 @@ defpgmodule Vtc.Ecto.Postgres.PgFramerate.Migrations do
     :ok
   end
 
+  @doc section: :migrations_functions
+  @doc """
+  Creates `rational_private.simplify(rat)` function that simplifies a rational. Used at
+  the end of every rational operation to avoid overflows.
+  """
+  @spec create_func_is_ntsc() :: :ok
+  def create_func_is_ntsc do
+    create_func =
+      Postgres.Utils.create_plpgsql_function(
+        :"#{function(:is_ntsc, Migration.repo())}",
+        args: [input: :framerate],
+        returns: :boolean,
+        body: """
+        RETURN (
+          (input).tags @> '{drop}'::framerate_tags[]
+          OR (input).tags @> '{non_drop}'::framerate_tags[]
+        );
+        """
+      )
+
+    :ok = Migration.execute(create_func)
+
+    :ok
```

Comment:
along the same lines as my comment above, this would just return `Migration.execute(create_func)`

Reply:
Removed unneeded matches in 462281841ad8396276633f67d28fb548aac2f1b7 per above reasoning.